sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/_base.py

```diff
@@ -100,6 +100,7 @@ def __init__(self, **kwargs: Any) -> None:
         self._storage_directory = kwargs.get('storage_directory', default_storage_directory)  # Storage path in which to store retry files.
         self._storage_retention_period = kwargs.get('storage_retention_period', 48 * 60 * 60)  # Retention period in seconds (default 48 hrs)
         self._timeout = kwargs.get('timeout', 10.0)  # networking timeout in seconds
+        self._distro_version = kwargs.get('distro_version', '')  # If set, indicates the exporter is instantiated via Azure monitor OpenTelemetry distro. Versions corresponds to distro version.
```

Comment:
We have version instead of a simple bool since we might use version in the future.

Reply:
Can we rework "_distro_version" to rely on a constant that the distro can pull from?